### ISSUE TITLE
Outlook's SMTP server support?

### ISSUE THREAD

**Issue by suntong** [s-gv/orangeforum#158]
As explained in [How to send email using Outlook's SMTP servers?](https://stackoverflow.com/questions/57783841/how-to-send-email-using-outlooks-smtp-servers), _"Golang smtp does not support LOGIN"_, which is what Outlook's using.

Ok to switch the Golang smtp to other packages that support Outlook out of the box?
If so, I can send in PR.

**Comment by s-gv** [comment]
Sure! Which package do you intend to use?

**Comment by suntong** [comment]
I've went through about 7 different ways, including https://github.com/Shopify/gomail, but found the https://github.com/wneessen/go-mail to be the best of all implementations, and it happens to be following idiomatic Go style and best practice, and It's only dependency is the Go Standard Library.

**Comment by s-gv** [comment]
Either sounds good to me! (I’d lean towards the one more actively
maintained, but please go ahead choose whichever you think is more suitable)

**Comment by suntong** [comment]
Yes, being actively maintained is my de-facto number one priority. 

- Shopify/gomail last update on Jul 29, 2022
- go-mail last update 3 weeks ago

**Comment by suntong** [comment]
Done, see 

https://github.com/suntong/orangeforum/compare/dev/email...suntong:orangeforum:dev/outlook?expand=1#diff-088d9f35d23a4347d221d71dd49b02b95001dff4abe637a40fe0bc04d502049c

It's built on top of the previous two branches, so if you don't agree with the previous two changes, this would be the starting point to merge...

**Comment by suntong** [comment]
If it's my own, I'd

- make the default `sendMail` to send html mail instead, so that the `if` condition for SMTPAuthLogin or SMTPAuthPlain can be embedded within itself, and
- I'll write another wrapper on top of it to send text in markdown as html mail.

**Comment by s-gv** [comment]
Not a big fan default HTML mail. I'd rather stick to plain text.

**Comment by suntong** [comment]
Yeah, thought so, that's why I left it unchanged.

**Comment by s-gv** [comment]
Cool. Can you send a PR with that? It doesn't seem to be here. Thx!

**Comment by suntong** [comment]
OK, https://github.com/s-gv/orangeforum/pull/161/

As commented, 

> It's built on top of my previous two branches

So to send PR for this, I need to send all previous changes. 

> with the latest bump version here, things are a bit complicated now

I've _manually_ resolved the conflicts.
